### PR TITLE
Update Obj-C and Swift wrappers. 

### PR DESCRIPTION
**Pull request by radetsky**
## Checklist

- [x] Example projects and code samples are up-to-date 
- [x] Changelog is updated (in case of notable or breaking changes)

### CODE REVIEW

File: docs/examples/swift/macOS-SPM/ThemisTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -5,8 +5,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/cossacklabs/themis",
       "state" : {
-        "revision" : "b573ad970462e5ccc9dbafe5d954e57dcc386ba8",
-        "version" : "0.14.9"
+        "revision" : "d925c3ba6bb13bf1914a58f2380dfac4698aa09e",
+        "version" : "0.15.0"
```
Comment (vixentael) [inline docs/examples/swift/macOS-SPM/ThemisTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:9]: let's call it  `0.15.1` to align with cocoapods

**Comment by vixentael** [comment]
@radetsky could you please also update tests that are failing? as we have changed the minimum version, the tests projects require a change as well.
Maybe it makes sense to merge this PR, and update tests in a separate PR

**Review by vixentael — APPROVED** [review]
